plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -353,14 +355,10 @@ function executeCLI($scriptName, &$args) {
 
 		switch ($command) {
 			case 'import':
-				$userName = array_shift($args);
-				$user = $userDao->getByUsername($userName);
+				if (!isset($user)) {
+					echo __('plugins.importexport.common.cliError') . "\n";
+					echo __('plugins.importexport.native.error.unknownUser', array('userName' => $userName)) . "\n\n";
```

Comment:
Is `$userName` still defined? It looks like this variable was removed.

Reply:
Fixed